apps/highlight/src/utils.ts

```diff
@@ -48,3 +48,12 @@ export function rpcError(
 export async function sleep(ms: number) {
   return new Promise(resolve => setTimeout(resolve, ms));
 }
+
+export function getSlug(text: string): string {
+  return text
```

Comment:
It's better to use this https://www.npmjs.com/package/lodash.kebabcase so we cover much more case (special character or others languages).